lxd/tokens.go

```diff
@@ -40,6 +80,20 @@ func autoRemoveExpiredTokens(ctx context.Context, s *state.State) {
 			}
 		}
 
+		err := s.DB.Cluster.Transaction(s.ShutdownCtx, func(ctx context.Context, tx *db.ClusterTx) error {
+			for _, expiredPendingTLSIdentity := range expiredPendingTLSIdentities {
+				err := cluster.DeleteIdentity(ctx, tx.Tx(), api.AuthenticationMethodTLS, expiredPendingTLSIdentity.Identifier)
+				if err != nil {
+					logger.Debug("Failed removing pending TLS identity", logger.Ctx{"err": err, "id": op.ID()})
+				}
+			}
+
+			return nil
+		})
+		if err != nil {
+			logger.Debug("Failed removing pending TLS identities", logger.Ctx{"err": err, "id": op.ID()})
+		}
```

Comment:
I am not too familiar with out logging standards, but I am wondering if this should be a `logger.Error`, to match the ones above on this same file. What do you think?

Reply:
It looks like the log messages that relate to collating the expired token operations are logged at error level and the log messages that related to deleting the expired token operations are logged at debug level. So this is consistent with what is already in the task. 

I'm happy to change it though :)